src/app/pages/Designathons/Designathon24/components/Schedule/index.jsx

```diff
@@ -1,33 +1,31 @@
-import { Section } from 'app/Symbols';
-import clsx from 'clsx';
-import cn from './Schedule.module.scss';
+import { Section } from "app/Symbols";
+import clsx from "clsx";
+import cn from "./Schedule.module.scss";
 
 function Schedule() {
 	return (
-		<div className={cn.container} id='s-schedule'>
+		<div className={cn.container} id="s-schedule">
 			<Section>
 				<h1 className={cn.title}>Schedule</h1>
 				<span className={cn.note}>
-					In-person events are marked with (IP) and are open to UCI
-					students only.
+					In-person events are marked with (IP) and are open to UCI students
+					only.
 				</span>
 				<h2>Friday - Feb. 24, 2023</h2>
```

Comment:
Thought [non-blocking, can revisit]: given how lengthy this file is getting, do you think we could parameterize the schedule components and apply the data from an array?

Reply:
Refactored the file and extracted out the events into the assets directory under `events.json`. 

There's some minor styling inconsistencies that didn't port over correctly and the location tags for Zoom and Discord aren't hyper-linked yet, but seeing as the hi-fi designs for the schedule haven't been finalized yet, I think it's in a good spot regarding refactoring.